app/routes.py

```diff
@@ -0,0 +1,273 @@
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return {'message': f'Customer {customer_id} was not found'}, 404
+    else:
+        response_body = request.get_json()
+    if not Customer.is_data_valid(response_body)[0]:
+        return Customer.is_data_valid(response_body)[1], 400
+    customer.name = response_body["name"]
+    customer.postal_code = response_body["postal_code"]
+    customer.phone = response_body["phone"]
+    db.session.commit()
+    return customer.to_dict(), 200
+
+
+# WV1 Delete one specific customer by ID. 404 error if customer of
+# specified ID is not found. 400 error if ID is not an integer.
+@customers_bp.route("/<customer_id>", methods=["DELETE"], strict_slashes=False)
+def delete_customer(customer_id):
+    if not Customer.is_int(customer_id):
+        return {'message': f'{customer_id} is not a valid customer id'}, 400
+    customer = Customer.query.get(customer_id)
+    if not customer:
+        return {'message': f'Customer {customer_id} was not found'}, 404
+    else:
+        rental_ids = Rental.query.with_entities(Rental.rental_id).filter(Rental.customer_id == int(customer_id))
+        for rental_id in rental_ids:
+            db.session.delete(Rental.query.get(rental_id)) # look into backref cascade? to make the tables do this automatically
+        db.session.commit()
+
+        db.session.delete(customer)
+        db.session.commit()
+        return {"id": customer.customer_id}, 200
+
+
+# Video routes
+
+#WV1: Get No Saved Videos Get Videos One Saved Video
+@videos_bp.route("", methods=["GET"], strict_slashes=False)
+def get_all_videos():
+    videos = Video.query.all()
+    response = [video.to_dict() for video in videos]
+    return jsonify(response), 200
+
+#WV1: Create Video, Must Contain Title/Release Date/Inventory
+@videos_bp.route("", methods=["POST"], strict_slashes=False)
+def post_all_videos():
+    request_body = request.get_json()
+    try:
+        new_video = Video(title = request_body["title"],
+            release_date = request_body["release_date"],
+            total_inventory = request_body["total_inventory"])
+    except KeyError:
+        if "title" not in request_body:
+            return jsonify({"details": "Request body must include title."}), 400
+        if "release_date" not in request_body:
+            return jsonify({"details": "Request body must include release_date."}), 400
+        if "total_inventory" not in request_body:
+            return jsonify({"details": "Request body must include total_inventory."}), 400
+    db.session.add(new_video)
+    db.session.commit()
+    return jsonify(new_video.to_dict()), 201
+
+#WV1: Invalid Video ID, Get Video Not Found, Get Video
+@videos_bp.route("/<video_id>", methods=["GET"])
+def get_one_video(video_id):
+    try:
+        video_id = int(video_id)
+        video = Video.query.get(video_id)
+    except ValueError:
+        return {}, 400
```

Comment:
Nice error checking!  I recommend creating a helper function to do this error checking to make it easy to add to update one video & delete one video.